doggystyle.css

```diff
@@ -0,0 +1,186 @@
+/*adoptadog stylesheet*/
+* {
+    box-sizing: border-box;
+}
+
+
+header {
+  width: 100%
+  border: none;
+  margin: none;
+  background-color: DarkCyan;
+  padding: 2%;
+}
+
+header {
+  display: flex;
+  flex-flow: row;
```

Comment:
flex direction is a better thing to use, make sure to specify `justify-content` and `align-items` if needed (you probably will want to use one, if not both)